p3-frontend/src/app/components/tags/tags.component.ts

```diff
@@ -44,29 +44,53 @@ export class TagsComponent implements OnInit {
   public tagsNames:Tag[] = [];
   public tags:Tag[] = [];
   public errorDetected:boolean = false;
+
+
   constructor(public projectService: ProjectService, public tagService: TagService,config: NgbModalConfig, private modalService: NgbModal) {
     config.backdrop = 'static';
     config.keyboard = false;
 
     this.filteredTags = this.tagCtrl.valueChanges.pipe(
       startWith(null),
+<<<<<<< HEAD
+      map((tagName:string|null) => tagName ? this._filter(tagName): this.tagsNames.slice()));
+  }
+
+
+  closeResult = '';
+
+  open(content:any) {
+    this.modalService.open(content);
+  }
+=======
```

Comment:
@Christian-Downs  please remove these merge conflict signals, save, commit, and then push again? Just remove what's between <<<HEAD and the ===